psl-core/src/main/java/org/linqs/psl/reasoner/admm/term/ADMMObjectiveTerm.java

```diff
@@ -196,21 +194,27 @@ public TermType getTermType() {
      * Modify the local variables to minimize this term (within the bounds of the step size).
      */
     public void minimize(float stepSize, float[] consensusValues) {
+        float weight = Float.POSITIVE_INFINITY;
+
```

Comment:
Remove this newline.
I want devs to mentally associate this entire block with weight initialization.